JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
 A value the DateBox displays.
 
 - [type](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#type)       
-Specifies whether the DateBox allows users to select only date, only time, or both.
+Specifies whether the DateBox allows users to select one of the following types:
+
+  - *"date"*    
+  Users can select the date from the calendar, or they can type in their own values.
```

Comment:
I noticed a warning icon is displayed if the date is not input in the correct format. Can we add "(in the required format)" at the end of this sentence?

`Users can select the date from the calendar, or they can type in their own values (in the required format).`

Reply:
Noted.

Reply:
Thanks. Note that I also changed "their own values" to "their own date value" in my suggestion above.